node/src/testing/multi_stage_test_reactor.rs

```diff
@@ -400,7 +400,9 @@ impl Reactor for MultiStageTestReactor {
 
                     let node_id = joiner_reactor.node_id();
                     effects.extend(joiner_reactor.into_validator_config().boxed().event(
-                        |validator_init_config| {
+                        |res_validator_init_config| {
+                            let validator_init_config = res_validator_init_config
+                                .unwrap_or_else(|err| panic!("failed {:?}", err));
```

Comment:
Might as well just `unwrap()`; "failed" doesn't add any information.

Reply:
Done in [`6d6256a` (#904)](https://github.com/CasperLabs/casper-node/pull/904/commits/6d6256afbfb52e3c3bcc4ea817550a3ce25cea36).